cfe_internal/update/simple_policy_update_v1.cf

```diff
@@ -0,0 +1,476 @@
+################################################################################
+#
+# DO NOT EDIT THIS FILE. All policy files prefixed with CFE_ and update_bins.cf
+# are maintained by CFEngine and its original state is required for internal
+# operations of CFEngine. If the file has been modified CFEngine’s upgrades
+# may require manual intervention. Contact CFEngine support if additional
+# information and/or recommendation is needed.
+#
+################################################################################
+#########################################################
+#
+# update_policy
+# - Cfengine policy update (masterfiles -> inputs)
+#
+#########################################################
+
+bundle agent simple_policy_update_v1_cfe_internal_update_policy
```

Comment:
I renamed cfe_internal_update_policy to simple_policy_update_v1_cfe_internal_update_policy  ( I was lazy and just prefixed it for now, better name welcome)